libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/trip/service/MapboxTripService.kt

```diff
@@ -25,7 +25,7 @@ import kotlinx.coroutines.channels.ReceiveChannel
  * @param terminateLambda called when [TripService] has stopped
```

Comment:
Let's remove the `TODO` ☝️

Reply:
This can be moved out of the `internal` package as it has the `internal` modifier now.

Reply:
fixed